src/js/components/Button/stories/RoutedButton.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 
-import { Box, Grommet, RoutedButton, Text } from 'grommet';
+import {
+  Box,
+  Grommet,
+  RoutedButton as GrommetRoutedButton,
+  Text,
+} from 'grommet';
 import { grommet } from 'grommet/themes';
 
-const RouteButton = () => (
+export const RoutedButton = () => (
   <Grommet theme={grommet}>
     <Box align="center" pad="large">
       <Text margin="medium" size="small">
         Note: RoutedButton will soon be deprecated
       </Text>
-      <RoutedButton label="Go" path="/" />
+      <GrommetRoutedButton label="Go" path="/" />
```

Comment:
Why was this change made?

Reply:
i needed to rename the imported `RoutedButton` from grommet in line 6